doc/guides/modules.rst

```diff
@@ -10,14 +10,342 @@ system those are called *modules*. These modules must be integrated with the
+individual roles are defined.
+
+**Administration:** Each Zephyr module shall have an administrator
+who is responsible for managing access to the module repository,
+for example, for adding individuals as Collaborators in the repository
+at the request of the module code owner. Module administrators are
+members of the Administrators' team, that is a group of project
+members with admin rights to Module Github repositories.
+
+**Code ownership:** Each module shall have a code owner. Module code
+owners will have the overall responsibility of the contents of a
+Zephyr module repository. In particular, a module code owner will
+
+* coordinate code reviewing in the module repository
+* be the default assignee in pull-requests against the repository's
+  main branch
+* request additional collaborators to be added to the repository, as
+  they see fit
+* regularly synchronize the module repository with its upstream
+  counterpart following the policies described in
+  :ref:`modules_synchronization`
+* be aware of security vulnerability issues in the external project
+  and update the module repository to include security fixes, as
+  soon as the fixes are available in the upstream code base
+* list any known security vulnerability issues, present in the
+  module code base, in Zephyr release notes.
+
+
+  .. note::
+
+     Module code owners are not required to be Zephyr
+     :ref:`Maintainers <project_roles>`.
+
+**Merging:** The Zephyr Release Engineering team has the right and the
+responsibility to merge pull requests in the main branch of a module
+repository.
+
+  .. note::
+
+     The merging of pull requests in module repositories shall be coupled
+     with the corresponding manifest file updates in the Zephyr main tree.
+
+
+Maintaining module code base
+============================
+
+Updates in the Zephyr main tree, for example, in public Zephyr APIs,
+may require patching a module code base. The responsibility for keeping
+the module code base up to date is shared between the **contributor** of
+such updates in Zephyr and the module **code owner**. In particular:
+
+* the contributor of the original changes in Zephyr is required to submit
+  the corresponding patches that are required in module repositories, to
+  ensure that Zephyr CI on the pull request with the original changes, as
+  well as the module integration testing are successful.
+
+* the module code owner has the overall responsibility for maintaining
+  the module code base in-synch with the Zephyr main tree, by doing, for
+  example, occassionally, more advance testing of the module code base.
+  The code onwer is required to fix issues in the module code base that
+  have not been caught by Zephyr pull request CI runs.
+
+  .. note::
+
+    The above polices apply on all Zephyr modules. However, modules which
+    Zephyr depends on, for example vendor HALs, are more crucial to be kept
+    up-to-date with Zephyr main tree.
```

Comment:
What is the value of this note? It seems like it could be deleted.

Reply:
I guess, stressing that some modules are more important to be kept in-sync, than other modules. Felt it had some value to keep the note.